modules/srvrlss-using-cluster-logging-find-knative-serving-logs.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies:
+//
+// cluster-logging-serverless.adoc
```

Comment:
"serverless/cluster-logging-serverless.adoc" - the assembly should be in the 'serverless' folder.